### DISCUSSION TITLE
create pb hooks route data backup

### DISCUSSION THREAD

**Discussion by AxelJPG** [pocketbase/pocketbase#5914] [Q&A]
It is possible to create a data backup from a custom path?

**Comment by ganigeorgiev** [comment]
Please don't open multiple questions for the same thing (_assuming that your question is related to https://github.com/pocketbase/pocketbase/discussions/5872_).

I haven't replied to your other question because I don't understand what actually you are asking (and the current post doesn't make it more clear either) but most likely the functionality you are looking for is not available out of the box.

You have better chance someone to assist you if you elaborate more in your original discussion about your actual use case and what you are trying to do.
